packages/cover-traffic/src/state.ts

```diff
@@ -17,17 +19,20 @@ export type OpenChannels = {
 
 export type State = {
   nodes: Record<string, PeerData>
+  // channels indexed by its channelId
   channels: Record<string, ChannelData>
   log: string[]
+  // currently non-closed cover traffic channels
   ctChannels: OpenChannels[]
   block: BN
+  // number of failed messages indexed by the base58-encoded string of node id.
   messageFails: Record<string, number>
 }
 
 export type PeerData = {
-  id: any //PeerId,
+  id: any //PeerId type, as implemented in IPFS
   pub: PublicKey
-  multiaddrs: any
+  multiaddrs: any // Multiaddress type
```

Comment:
Same as above, there is a type for this.

Reply:
Noted.